deepchecks/core/serialization/common.py

```diff
@@ -446,3 +449,134 @@ def join(l: t.List[A], item: B) -> t.Iterator[t.Union[A, B]]:
         yield el
         if index != list_len:
             yield item
+
+
+def figure_to_html(figure: BaseFigure) -> str:
+    div_id = f'deepchecks-{get_random_string(n=25)}'
+    data = figure.to_json()
+    script = FIGURE_CREATION_SCRIPT.replace('%container-id', div_id).replace('%figure-data', data)
+    return (
+        f'<div><div id="{div_id}"></div>'
+        f'<script id="deepchecks-plot-initializer" type="text/javascript">{script}</script></div>'
+    )
+
+
+FIGURE_CREATION_SCRIPT = r"""
```

Comment:
where is this script coming from?

Reply:
I derived it from a script that is used by plotly library

Reply:
there is no way to get it using the library like `from plotly.offline.offline import get_plotlyjs`?

Reply:
no